pyscriptjs/src/plugin.ts

```diff
@@ -266,3 +266,72 @@ export function define_custom_element(tag: string, pyElementClass: PyElementClas
 
     customElements.define(tag, ProxyCustomElement);
 }
+
+// Members of py-config in plug that we want to validate must be one of these types
+type BaseConfigObject = string | boolean | number;
```

Comment:
Do you want to allow `undefined` here perhaps?

Reply:
I only ask since you typecast it explicitly here: https://github.com/JeffersGlass/pyscript/blob/3fa082df26f1a1d702779e1ac93a875426e71313/pyscriptjs/src/plugin.ts#L298

and check for `undefined` in the very next line.

Reply:
Ahhh I see now - sorry was writing the merge as you posted this.